### PR TITLE
Fix responsive lateral menu

### PR DESCRIPTION
**Pull request by DiegoCoronaBarragan**
# Description

I use styles to adjust the lateral menu in mobile view.

## Screenshots

image

### CODE REVIEW

**Review by claunicole — APPROVED** [review]

File: app/assets/stylesheets/pages/_main.scss [hunk 1]
```diff
@@ -751,5 +749,54 @@
...
+  }
+}
+
+@media (min-width: 760px) {
+  @media (max-width: 850px) {
+    
+    .main_menu_left {
+      
+      .navbar__container {
+
```
Comment (YilKanda) [inline app/assets/stylesheets/pages/_main.scss:794]: Remove unnecessary spaces

File: app/javascript/navbarService.js [hunk 2]
```diff
@@ -1,13 +1,20 @@
 const openNav = () => {
-  document.getElementById("mySidenavService").style.width = "12%";
-  document.getElementById("mainService").style.marginLeft = "96%";
-  // document.getElementById("mainService").style.background = "none";
-  document.getElementById("logoService").style.display = "none";
-
+  if (window.innerWidth >= 1080) {
+    document.getElementById("mySidenavService").style.width = "12%";
+    document.getElementById("mainService").style.marginLeft = "96%";
+    // document.getElementById("mainService").style.background = "none";
```
Comment (YilKanda) [inline app/javascript/navbarService.js:5]: Code that you do not use delete it.

File: app/javascript/navbarService.js [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
+    document.getElementById("logoService").style.display = "none";
+  }else {
+    document.getElementById("mySidenavService").style.width = "100%";
+    document.getElementById("mainService").style.marginLeft = "100%";
+    // document.getElementById("mainService").style.background = "none";
```
Comment (YilKanda) [inline app/javascript/navbarService.js:10]: Code that you do not use delete it.

**Review by YilKanda — CHANGES_REQUESTED** [review]
I have a question, why does the menu icon appear as vertical lines, instead of horizontal lines? Try to use standard icons that most users understand what they mean.

**Review by YilKanda — APPROVED** [review]

**Review by oscaregr — APPROVED** [review]

**Pull request merged**